components/vf-masthead/vf-masthead.config.yml

```diff
@@ -12,23 +12,31 @@ variants:
   - name: default
   - name: primary
     context:
-      mastheadThemeVariant: vf-masthead-theme--primary
+      Primary: true
   - name: secondary
     context:
-      mastheadThemeVariant: vf-masthead-theme--secondary
+      Secondary: true
   - name: tertiary
     context:
-      mastheadThemeVariant: vf-masthead-theme--tertiary
-  - name: with background image
+      Tertiary: true
+
+  - name: title block
     context:
-      mastheadImageClass: vf-masthead--has-image
-      mastheadImage: "../../assets/vf-masthead/assets/group-bg_2d4155.png"
-  - name: with title block
+      hasTitleBlock: true
+
+  - name: title block (primary)
     context:
-      hasTitleBlock: vf-masthead--with-title-block
+      hasTitleBlock: true
+      Primary: true
 
-  - name: kitchen sink
+  - name: image
     context:
-      mastheadImageClass: vf-masthead--has-image
-      mastheadImage: "../../assets/vf-masthead/assets/group-bg_2d4155.png"
-      hasTitleBlock: vf-masthead--with-title-block
+      mastheadImageClass: true
+      mastheadImage: "http://localhost:3003/vf-core/assets/vf-masthead/assets/group-bg_2d4155.png"
```

Comment:
I know this is in progress, but if we put the images on https://www.embl.org/files we can make use of the CDN and not have to worry about paths or storing the demonstration images in version control.